queries.sql

```diff
@@ -1,3 +1,19 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
```

Comment:
- [ ] according to the requirements your query should be able to find all animals which names ends in "mon" right now your query finds only animals called Luna.

Reply:
thank you.